.github/workflows/main.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - master
-      - f/WEP-153-comments-listing-route
 
```

Comment:
Think you should try: git checkout master .github/workflows/main.yml

Reply:
@faebebin I guess you are right that no need to have in master another branch will be deployed anytime we merge anything to master